Core/LocalAdmin.cs

```diff
@@ -34,6 +36,12 @@ internal class LocalAdmin
         private ushort gamePort;
         private bool exit;
 
+        static LocalAdmin()
+        {
+            // Installing the handler once
+            NativeSignalHandler.Setup();
```

Comment:
`if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) NativeSignalHandler.Setup();`, because you use a Windows-only library there (Kernel32).

You can try using Mono.Posix on Linux https://www.nuget.org/packages/Mono.Posix.NETStandard/ .